Documentation/DeveloperGuide/README.md

```diff
@@ -8,14 +8,22 @@ Follow these steps to run the addon directly from source so that your changes wi
 
 # Running on windows
 
+The project consists of two parts:
+- a .NET Core project which handles all network interaction.
+- maxScripts which integrate the .NET project and create the user interface
+
+This guide covers how to run maxScript and build .NET in Visual Studio Code.
+Template task.json and launch.json files are included in the repository.
+
 **Visual Studio Code**
 
-Open the project in VS Code.
+Clone the repository.\
+Open the repository as workspace in VS Code.
 
 To deploy code directly from VS Code to 3ds Max follow this tutorial:
 https://walterlow.com/setting-up-visual-studio-code-as-an-editor-for-3ds-maxscript/
 
-Use this build task to run the project:
+Change **"C:\\PATH_TO\\MXSPyCOM.exe"** in task.json to the correct file location:
 ```json
```

Comment:
this snippet is no longer needed because it's in the repository

Reply:
This is still needed for autocompletion